garak/langproviders/base.py

```diff
@@ -128,14 +133,94 @@ def is_meaning_string(text: str) -> bool:
 # To be `Configurable` the root object must meet the standard type search criteria
 # { langproviders:
 #     "local": { # model_type
-#       "language": "<from>-<to>"
+#       "language": "<from>,<to>"
 #       "name": "model/name" # model_name
 #       "hf_args": {} # or any other translator specific values for the model_type
 #     }
 # }
 from garak.configurable import Configurable
 
 
+class TranslationCache:
+    def __init__(self, provider: "LangProvider"):
+        if not hasattr(provider, "model_type"):
+            return None  # providers without a model_type do not have a cache
```

Comment:
why? is this expressing a non-explicit convention in LangProvider naming? if so would prefer this to be expressed explicitly